File: FavoDeMel-API/src/FavoDeMel.Domain/Entities/Comanda.cs

using FavoDeMel.Infrastructure.Data;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;

namespace FavoDeMel.Domain.Entities
{
    public class Comanda : Entity
    {
        public int NumeroMesa { get; set; }
        public Guid GarcomId { get; set; }
        public int Situacao { get; set; }
        public IEnumerable<Pedido> Pedidos { get; set; }
    }
}

File: FavoDeMel-UI/src/app/shared/models/comanda.ts
import { SituacaoComanda } from '../enums/situacao-comanda.enum';
import { Pedido } from './pedido';

export interface Comanda {
  id: string,
  situacao: SituacaoComanda
  pedidos: Pedido[],
  numeroMesa: number,
  garcomId: string,
}